Added analogy intro & extra questions

kravens opened the pull request:
Thunderbolt analogy to Lightning Network payment.
More beginner question.

kravens committed:
Added analogy intro & extra questions

Thunderbolt analogy to Lightning Network payment.
More beginner question.

GlenCooper reviewed (approved):
LGTM

nopara73 commented on 01-what-is.asciidoc:
```diff
@@ -2,4 +2,24 @@
+
+== Why use the Lightning Network?
+
+A second layer protocol can magnify the amount of transactions possible on the settlement layer, so that the base layer will not be overloaded with transactions as Bitcoin gains more active (transacting) participants. 
```
> as Bitcoin gains more active (transacting) participants. 

I think it's uneccessarily complex phrasing.

kravens replied:
Yeah, I tend to use very technical terms. What would you suggest here?
“As Bitcoin adoption grows in the world” perhaps.

nopara73 replied:
ACK

nopara73 commented on 01-what-is.asciidoc:
```diff
@@ -2,4 +2,24 @@
+
+A second layer protocol can magnify the amount of transactions possible on the settlement layer, so that the base layer will not be overloaded with transactions as Bitcoin gains more active (transacting) participants. 
+Instead small value payments can be instantly transmitted in a Lightning Channel, or a chain of channels through the Lightning Network.
+As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from eachother.
```
typo

```suggestion
As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from each other.
```

nopara73 replied:
Also I'm not entirely sure it's correct.

nopara73 commented on 01-what-is.asciidoc:
```diff
@@ -2,4 +2,24 @@
+== How to use the Lightning Network?
+
+Many interoperable implementations are under active development using the BOLT specifications. 
+It can be run as a Mobile Wallet App on any smartphone, a desktop light-client or routing-node.
```
Why the capitalization?

kravens replied:
Lightning Network was capitalized in the title, BOLT is an abreviation. Mobile wallet app should be lowercase indeed!

nopara73 replied:
Yes, I didn't argue with the LN capitalization.

```suggestion
It can be run as a mobile application, a desktop light-client or routing-node.
```

nopara73 commented on 01-what-is.asciidoc:
```diff
@@ -2,4 +2,24 @@
+Then a bright flash of light followed by a loud bang occurs as the current dissipated in the path of ionized air channels creates a hot plasma.
+The Lightning Network is named after this natural phenomenon, where the voltage differences are channel balances and the current is a stream of bitcoin of which ownership is transmitted through those channels.
+When lightning strikes the ground, the potential differences are settled at the cost of dissipating energy and making a loud bang.
+In the Lightning Network this happens when payment channels close, the final state of the channel is transmitted to the Bitcoin network with a fee for the miners to include it in the next block.
```
I learned physics in my native language, therefore I'm having a really difficult time to follow the analogy, due to not being familiar with the terminology. I admit it's a badass analogy, but making readers stop reading from the very first paragraph may not be the best way to go.  
The counter argument count be is that, at this point everyone is paying full attention and whoever chews through this first paragraph will find the following parts easier.  
Anyhow, not against it, but I think it worth a discussion.

kravens replied:
I will make a nice visual diagram to go with it, so that the concept is clear even without having learned the physics jargon.

super-e replied:
Agree with Adam, you risk to lose 90% of readers at the first paragraph :) I have a PhD in physics but still I'm not sure what the point was.

bitcoina replied:
I like the analogy, but agree with some of the comments here that its too complex and risks losing readers straight away.   It may be better to start with language such as "lightning is a way to improve upon the bitcoin protocol with additional functionality that increases transaction throughput and reduces settlement time. "

super-e commented:
Why not starting the "why" with the explanation of the problems the LN tries to solve? For example mentioning problems with scalability, confirmation time, fees cost, etc ?

elkimek commented:
How about to start with this: https://bitcointalk.org/index.php?topic=2500.msg34211#msg34211 ? :)

kravens committed:
Update with new first paragraph

Included comments from @nopara73, @bitcoina, @super-e and @elkimek.

elkimek reviewed (approved):
Looks good to me :)

kravens committed:
Sketch of Thunderbolt analogy

First version of diagram to help explain the Lightning Network

kravens committed:
Display figure inline

Learned how to do this: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#images
Placed figure with a Chain of Lightning channels.

renepickhardt commented:
> How about to start with this: https://bitcointalk.org/index.php?topic=2500.msg34211#msg34211 ? :)

while I admire Hal for the pioneering work that he did he was obviously wrong on that one. How I understand his post he expected that banks would basically create what we would today call side chains in order to scale bitcoin. I think this is not a good motivation for lightning. 
I would much rather prefer (and will add now) this resource: https://www.metzdowd.com/pipermail/cryptography/2008-November/014814.html which was the first reply. Not sure if random lucky guess or with great knowledge precisely on point.

kravens committed:
New starting quote

aantonop commented:
Too early to be writing the introduction. Typically I write the introduction paragraph LAST, as it needs to reflect the things we learn in each chapter. The first chapter introduction is probably the last thing we write/edit in the book
